addons/l10n_rs_edi/wizard/account_move_send.py

```diff
@@ -0,0 +1,106 @@
+
+
+class AccountMoveSend(models.TransientModel):
+    _inherit = 'account.move.send'
+
+    l10n_rs_edi_send_enable = fields.Boolean(compute="_compute_l10n_rs_edi_send_enable")
+    l10n_rs_edi_send_readonly = fields.Boolean(compute="_compute_l10n_rs_edi_send_readonly")
+    l10n_rs_edi_send_checkbox = fields.Boolean(
+        string="eFaktura",
+        compute="_compute_l10n_rs_edi_send_checkbox",
+        store=True, readonly=False,
+        help="Send the E-Invoice to Government via eFaktura"
+        )
+    l10n_rs_edi_send_cir_checkbox = fields.Boolean(
+        string="Send To CIR",
+        default=False,
+        help="Send to Central Invoice Register(For B2G and the public sector)"
+    )
+
+    @api.depends('move_ids.l10n_rs_edi_state')
+    def _compute_l10n_rs_edi_send_readonly(self):
+        for wizard in self:
+            wizard.l10n_rs_edi_send_readonly = all(move.l10n_rs_edi_state == 'sent' for move in wizard.move_ids)
+
+    @api.depends('move_ids.l10n_rs_edi_is_eligible')
+    def _compute_l10n_rs_edi_send_enable(self):
+        for wizard in self:
+            wizard.l10n_rs_edi_send_enable = any(move.l10n_rs_edi_is_eligible for move in wizard.move_ids)
+
+    @api.depends('l10n_rs_edi_send_readonly')
+    def _compute_l10n_rs_edi_send_checkbox(self):
+        for wizard in self:
+            wizard.l10n_rs_edi_send_checkbox = not wizard.l10n_rs_edi_send_readonly
+
+    @api.onchange('l10n_rs_edi_send_checkbox')
+    def _onchange_l10n_rs_edi_send_checkbox(self):
+        if not self.l10n_rs_edi_send_checkbox:
+            self.l10n_rs_edi_send_cir_checkbox = False
+
+    def _get_wizard_values(self):
+        # EXTENDS 'account'
+        vals = super()._get_wizard_values()
+        vals['l10n_rs_edi_send'] = self.l10n_rs_edi_send_checkbox
+        return vals
+
+    @api.model
+    def _get_wizard_vals_restrict_to(self, only_options):
+        # EXTENDS 'account'
+        values = super()._get_wizard_vals_restrict_to(only_options)
+        return {
+            'l10n_rs_edi_send_checkbox': False,
+            **values,
+        }
+
+    @api.model
+    def _need_invoice_document(self, invoice):
+        # EXTENDS 'account'
+        return super()._need_invoice_document(invoice) and not invoice.l10n_rs_edi_attachment_id
+
+    @api.model
+    def _get_invoice_extra_attachments(self, invoice):
+        # EXTENDS 'account'
+        return super()._get_invoice_extra_attachments(invoice) + invoice.l10n_rs_edi_attachment_id
+
+    @api.model
+    def _call_web_service_after_invoice_pdf_render(self, invoices_data):
+        # EXTENDS 'account'
+        super()._call_web_service_after_invoice_pdf_render(invoices_data)
+        for invoice, invoice_data in invoices_data.items():
+            # Not all invoices may need EDI.
+            if not invoice_data.get("l10n_rs_edi_send") or not invoice.l10n_rs_edi_is_eligible:
+                continue
+            self._check_l10n_rs_edi_required_info(invoice)
```

Comment:
You must not raise anything in this part of the code. Otherwise, you will trigger rollback and you don't know if there are multiple web-services used together or not